src/main/webapp/user.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Get the user @param user from the user servlet.
+ * @param {String} user
+ * @return a user object
+ */
+function getUser(user) {
+  console.log('entering get user function/n');
+  fetch('/user?user=' + user)
```

Comment:
Consider using [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) in the future if you plan on adding more parameters.

Reply:
I changed this in a future PR